internal/eval/eval_test.go

```diff
@@ -1070,84 +1071,84 @@ func testEvalJSONSET(t *testing.T, store *dstore.Store) {
 }
 
 func testEvalJSONNUMMULTBY(t *testing.T, store *dstore.Store) {
-    tests := map[string]evalTestCase{
```

Comment:
seems like linting issues got fixed when my editor ran format. seems like we don't enforce go fmt in CI